beacon-chain/core/state/spectest/slot_processing_test.go

```diff
@@ -63,6 +66,27 @@ func runSlotProcessingTests(t *testing.T, config string) {
 				t.Log(diff)
 				t.Fatal("Post state does not match expected")
 			}
+
+			// Process slots and epoch with optimizations.
+			app := cli.NewApp()
+			set := flag.NewFlagSet("optimize-process-epoch", 0)
+			set.Bool(featureconfig.OptimizeProcessEpoch.Name, true, "optimize process epoch")
+			ctx := cli.NewContext(app, set, nil)
+			featureconfig.ConfigureBeaconChain(ctx)
```

Comment:
This isn't the best way to enable that flag.

You should do something like
```go
f := featureConfig.Get()
f.OptimizeProcessEpoch = true
featureConfig.Init(f)
```
Also shouldn't this be enabled at the beginning of the test?

Reply:
Thanks, it's in the middle because we test twice. One with optimizations and one without

it goes like this:
```go
func runSlotProcessingTests() {
preState := readYaml()
preStateCopy := proto.Clone(preState)

postState := procesSlots(preState)
// check postState

f := featureConfig.Get()
f.OptimizeProcessEpoch = true
featureConfig.Init(f)

postState := procesSlots(preStateCopy)
// check postState
}
```